aws/resource_aws_eip_association.go

```diff
@@ -157,7 +157,30 @@ func resourceAwsEipAssociationRead(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 
-	response, err := conn.DescribeAddresses(request)
+	var response *ec2.DescribeAddressesOutput
+	err = resource.Retry(2*time.Minute, func() *resource.RetryError {
```

Comment:
Nit: We should create a constant for this time value (check out `aws/internal/service/ec2/waiter.go`). See also: https://github.com/hashicorp/terraform-provider-aws/issues/16753

Reply:
Yes, I added a general PropagationTimeout for EC2-related resources there (as of now there's only `InstanceAttributePropagationTimeout`)